recipes/jupyter-docx-bundler/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "jupyter-docx-bundler" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9457377f8d25dcde1a4de241f31ddc31d18d503f00ccf02823393729196584a1
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

This looks like it could be `noarch`?
```suggestion
  noarch: python
```

---

I thought about that too. I tried it in a PR the package repo (https://github.com/m-rossi/jupyter-docx-bundler/pull/26) and it seemed to have worked, even the linking scripts.

However the conda-forge docs are pretty clear regarding this topic: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

> In order to qualify as a noarch python package, all of the following criteria must be fulfilled:
> * [...]
> * No post-link or pre-link or pre-unlink scripts
> * [...]

---

Ah, I forgot about that limitation. In that case, LGTM :+1: